benchmarks/bare/Makefile

```diff
@@ -1,10 +1,10 @@
 bmarkdir := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 
 TARGET := riscv64-unknown-elf
-CFLAGS := -mcmodel=medany -std=gnu99 -O2 -fno-common -fno-builtin-printf -Wall
+CFLAGS := -mcmodel=medany -std=gnu99 -O2 -fno-common -fno-builtin-printf -Wall -I $(bmarkdir)
```

Comment:
It would be better to move `encoding.h` to the shared `src` directory.  Then this include path would not be needed.